nuget/nunit.runners.xamarin.nuspec

```diff
@@ -23,7 +23,7 @@ Supported Xamarin platforms:
 
         <dependencies>
             <group>
-                <dependency id="nunit" version="[3.6.1]" />
```

Comment:
We should keep the square brackets here. nunit.xamarin _should_ currently depend on a specific version of the NUnit framework. In your particular instance, upgrading NUnit alone worked, but that was more by coincidence, than a supported method!